src/main/java/org/springframework/integration/dsl/core/PollerSpec.java

```diff
@@ -109,9 +110,21 @@ public PollerSpec advice(Advice... advice) {
 	 * provided {@code PlatformTransactionManager} for the {@code pollingTask}.
 	 * @param transactionManager the {@link PlatformTransactionManager} to use.
 	 * @return the spec.
+	 * @deprecated - since 1.2 use {@link #transactional(TransactionInterceptor)}
 	 */
+	@Deprecated
```

Comment:
I don't think so. 
There is no problems with this method.
It is a good choice to be based on just a provided `PlatformTransactionManager`.
I also see one more choice as a argumentless `.transactional()`, where I don't see reason to require `new TransactionInterceptorBuilder().build()` from end-user where he/she would like to rely on the default for everything.
WDYT?